docker-mixin/dashboards/docker.libsonnet

```diff
@@ -35,18 +35,19 @@ local ds_template = {
   },
   hide: 0,
   label: 'Data Source',
-  name: 'datasource',
+  name: 'prometheus_datasource',
   options: [],
   query: 'prometheus',
   refresh: 1,
-  regex: '',
+  regex: '(?!grafanacloud-usage|grafanacloud-ml-metrics).+',
```

Comment:
This seems grafana cloud specific, and should probably not be done in this lib? I think this is already taken care of [here for cloud](https://github.com/grafana/cloud-onboarding/blob/7588a0520269d704d2cdb251c2cf48f1728d9c95/integrations/util.libsonnet#L15).

Reply:
Thanks @RutgerKe ! You are right - I will remove this.